homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -324,6 +332,7 @@ class PlenticoreSensorEntityDescription(
         name="Home Consumption Day",
```

Comment:
For reviewer: The following entities are measurement of the energy during various intervals.

Reply:
To understand it correctly...
This entity shows the kWh of the home consumption per day, correct?
Will it reset to 0 at midnight?

Reply:
Yes, for example like this:

![grafik](https://github.com/home-assistant/core/assets/6886353/3e6357cc-3e32-4cec-a777-c0f0d9e5e594)